addons/mail/static/src/core/public_web/messaging_menu.xml

```diff
@@ -32,7 +32,11 @@
                         <t t-else="" t-esc="thread.displayName"/>
                     </t>
                     <t t-set-slot="body">
-                        <t t-esc="message?.inlineBody or message?.subtype_description" t-att-class="{ 'opacity-75': message?.isEmpty }"/>
+                        <t t-if="message?.hasOnlyAttachments">
+                            <i class="fa me-1" t-att-class="message.messagePreviewIcon"/>
+                            <t t-out="message.messagePreviewText" />
+                        </t>
+                        <span t-else="" t-esc="message?.inlineBody or message?.subtype_description" t-att-class="{ 'opacity-75': message?.isEmpty }"/>
```

Comment:
`t-else=""` can just be `message.previewText` rather than `?.inlineBody or message?.subtype_description`.